arangod/RocksDBEngine/RocksDBTransactionCollection.cpp

```diff
@@ -63,6 +63,7 @@ RocksDBTransactionCollection::RocksDBTransactionCollection(
 
 RocksDBTransactionCollection::~RocksDBTransactionCollection() {
   try {
+    // cppcheck-suppress virtualCallInConstructor
     releaseUsage();
```

Comment:
I guess you could also do
```suggestion
RocksDBTransactionCollection::releaseUsage();
```
instead of the suppression. I don't care, though.